.circleci/load_env.sh

```diff
@@ -2,10 +2,10 @@
 
 echo 'export GITHUB_REPO=GladysAssistant/Gladys' >>$BASH_ENV
 echo 'export IMAGE=gladys' >>$BASH_ENV
-echo 'export REGISTRY=gladysassistant' >>$BASH_ENV
+echo 'export REGISTRY="${DOCKERHUB_REPO}"' >>$BASH_ENV
```

Comment:
I'm not sure this is working. $DOCKERHUB_REPO is already used and is "gladysassistant/gladys", not "gladysassistant"

Reply:
You are right, i need to clean env and parameter in ci flow + add information in dev documentation

In that case IMAGE_ID is wrong

Reply:
So should I rollback for now, waiting @vonox to fix it in another PR?

Reply:
Yes I think Alex

Reply:
Ok, I'll rollback both modified in circleci folder (config.yml / load_env.sh).
@VonOx I hope you'll fix it ASAP :)

Reply:
Yep, I will do this week